### ISSUE TITLE
Some translations are not reflected

### ISSUE THREAD

**Issue by yumunet** [PrismLauncher/PrismLauncher#3056]
### Operating System

Windows

### Version of Prism Launcher

9.1

### Version of Qt

Qt 6.7.3

### Description of bug

The translation is not reflected in the "Download mods", the dialog when "Check for update" is pressed, etc.

This is how it looks. (in Japanese)

![bar](https://github.com/user-attachments/assets/dd854c47-5f54-43a2-b3b6-27f7aa2583a9)

![update dialog](https://github.com/user-attachments/assets/58464792-427e-4073-8e19-574aca7879b1)

It is not untranslated, and the same is true in Deutsch, Indonesia, etc., where Completense is 100%.
It was translated until about two weeks ago.

I would like to see the translations reflected in the stable 9.1 version.

### Steps to reproduce

1. Change language from settings
2. Edit instance
3. Open "Mods"
4. Download an older version of anything mod and check for updates

### Suspected cause

First, I understand that Prism translations are hosted on Weblate and are regularly reflected in Prism itself via a dedicated server.

This commit seems to have removed "location"s in "<name>ModUpdateDialog</name>" and other places.
I believe this is the direct cause.
https://github.com/PrismLauncher/Translations/commit/a9bd3648aa28f6baba07d9ab9c887a49bce5d50e
(It's slow to show up on GitHub, so you might want to clone it locally.)

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by Trial97** [comment]
@TheKodeToad what is the possibility of this being caused by: https://github.com/PrismLauncher/PrismLauncher/pull/1588 ?

**Comment by TheKodeToad** [comment]
I changed the capitalisation to be consistent, it certainly should not affect 9.1 so this must be a flaw with our translations system

**Comment by Ryex** [comment]
Changed capitalization should be treated as a new string.

**Comment by yumunet** [comment]
Thank you all for your help.
I have looked into it carefully once.

I looked at the code in the Translations repository and it seems that the translations are only updated by Qt's [lupdate](https://doc.qt.io/qt-6/linguist-lupdate.html) command.
https://github.com/PrismLauncher/Translations/blob/master/update.sh#L20

And this lupdate seems to add `type="vanished"` to the `translation` tag of translations that are only in older versions, which are excluded when building to .qm file by [lrelease](https://doc.qt.io/qt-6/linguist-lrelease.html).
Therefore, the current system always replaces the translations for the latest source code.
This means that if an existing string is changed due to UI changes, etc., the translation of that portion will not be reflected until the UI change is released.

I think the translations need to be separated or merged between the latest version and the stable version (merging is more practical).
Specifically, I think it is to remove the `type="vanished"` of translations that are only available in the stable version after the lupdate run (download the source code of the stable version from GitHub and compare the translations).

I have also created an issue in the Translations repository about this issue.
https://github.com/PrismLauncher/Translations/issues/20
